src/utils/bootstrapUtils.js

```diff
@@ -15,12 +15,18 @@ function curry(fn) {
   };
 }
 
-export function prefix(props, variant) {
+export function prefix(props, context = {}, variant) {
```

Comment:
it'd only be relevant if only `props` were passed in, but that would make no sense. i think you can drop the `= {}` here.